[2024-10-02T19:03:55.608+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T19:03:55.804+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-08-11T00:00:00+00:00 [queued]>
[2024-10-02T19:03:55.812+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-08-11T00:00:00+00:00 [queued]>
[2024-10-02T19:03:55.813+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T19:03:55.823+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): carregar> on 2024-08-11 00:00:00+00:00
[2024-10-02T19:03:55.830+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=8055) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T19:03:55.831+0000] {standard_task_runner.py:63} INFO - Started process 8062 to run task
[2024-10-02T19:03:55.831+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'carregar', 'scheduled__2024-08-11T00:00:00+00:00', '--job-id', '1877', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpc92ec15f']
[2024-10-02T19:03:55.833+0000] {standard_task_runner.py:91} INFO - Job 1877: Subtask carregar
[2024-10-02T19:03:55.872+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-08-11T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T19:03:55.947+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='carregar' AIRFLOW_CTX_EXECUTION_DATE='2024-08-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-11T00:00:00+00:00'
[2024-10-02T19:03:55.948+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T19:03:55.998+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T19:03:55.998+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T19:03:56.006+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=carregar, run_id=scheduled__2024-08-11T00:00:00+00:00, execution_date=20240811T000000, start_date=20241002T190355, end_date=20241002T190356
[2024-10-02T19:03:56.044+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T19:03:56.057+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T19:03:56.059+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
